src/main/resources/organisationSample.json

```diff
@@ -3,20 +3,20 @@
     {
       "organisationIdentifier": "1",
       "status": "ACTIVE",
-      "lastUpdated": "2023-11-20T15:51:33.046Z",
+      "organisationLastUpdated": "2023-11-20T15:51:33.046Z",
```

Comment:
The API still returns `lastUpdated` because that field belongs to the `Organisation` object. I think a few others have been caught out by this. The only thing you should need to map `lastUpdate` from the API response to `organisationLastUpdate`.

Reply:
corrected